microsoft/react-native-test-app
# fix: Open last component from previous session

### Description

Adds a toggle to reopen the component that was last opened in previous session.

### Platforms affected


### Test plan

1. Build and run Example app
2. Note that we are seeing a blank window on desktop, and the component list on mobile
3. Enable _Remember last opened component_ (see screenshots below)
4. Open any component
5. Restart Example app
6. The component that was opened last session should now be reopened
7. Disable _Remember last opened component_
8. Restart Example app
9. We should now see either a blank window or the component list
10. Repeat steps 1-9 for all platforms

| Android | iOS | macOS | Windows |
|:-:|:-:|:-:|:-:|
| ![image](https://user-images.githubusercontent.com/4123478/102710441-9e288680-42b2-11eb-9847-f124583f9260.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2020-12-20 at 11 00 04](https://user-images.githubusercontent.com/4123478/102710458-bc8e8200-42b2-11eb-8964-23330b3a2d53.png) | <img width="1440" alt="image" src="https://user-images.githubusercontent.com/4123478/102698794-5b2dcb00-4240-11eb-9699-c952fd6aea4c.png"> | ![image](https://user-images.githubusercontent.com/4123478/102698222-23bd1f80-423c-11eb-8c0d-62ddfbdd6a87.png) |
